tests/plugins/find_and_replace.test.ts

```diff
@@ -150,6 +150,24 @@ describe("basic search", () => {
     expect(matches).toHaveLength(2);
     expect(matchIndex).toStrictEqual(0);
   });
+
+  test("hidden cells should not be included in match", () => {
+    model.dispatch("UPDATE_SEARCH", { toSearch: "1", searchOptions });
+    let matches = model.getters.getSearchMatches();
+    let matchIndex = model.getters.getCurrentSelectedMatchIndex();
+    expect(matches).toHaveLength(4);
+    expect(matchIndex).toStrictEqual(0);
+    model.dispatch("HIDE_COLUMNS_ROWS", {
+      dimension: "ROW",
```

Comment:
use the helper hideRow

Reply:
Changes done, can you please review this PR?